internal/pgmonitor/postgres.go

```diff
@@ -93,20 +93,31 @@ func EnableExporterInPostgreSQL(ctx context.Context, exec postgres.Executor,
 
 	stdout, stderr, err := exec.ExecInAllDatabases(ctx,
 		strings.Join([]string{
+			// Quiet NOTICE messages from IF EXISTS statements.
+			// - https://www.postgresql.org/docs/current/runtime-config-client.html
+			`SET client_min_messages = WARNING;`,
```

Comment:
From the name, I assume this is only valid for the client, which is why you have to set it both here and on L119 and also why it doesn't have to be undone after, yeah?

Reply:
Yes. It affects only one/current connection and has no effect after the connection closes.